README.md

```diff
@@ -41,6 +41,10 @@ Head to the address it gives you to check if your changes have worked. More info
 ## Licences
 Third party licenses have been used, see Licenses, Apache 2.0 and OpenLDAP, kdbldap.so   
 ## Release Notes
+- **5.0.0, Dec 2023**
+  * Updating TorQ to support AWS Finspace when using the new [TorQ-Finspace-Starter-Pack](https://dataintellecttech.github.io/TorQ-Finspace-Starter-Pack/)
+  * Fixes to heartbeat disabling logic
+  * log output can now be json file format: add '-jsonlogs true' flag on startup
```

Comment:
I think there might be a few non-FinSpace related changes to mention here since last release as well. Looks like these commits have all been added since 4.4.0 https://github.com/DataIntellectTech/TorQ/compare/v4.4.0...master